src/main/resources/i18n/messages.properties

```diff
@@ -960,21 +960,21 @@ teammatchmaking.match.starting = Match launching...
 teammatchmaking.match.cancelled = Match cancelled\!
 teammatchmaking.notification.matchFound.title = Match found
 teammatchmaking.notification.matchFound.message = A match was found and will launch soon
-teammatchmaking.notification.inviteAlreadyInQueue.title = You are already in a queue
-teammatchmaking.notification.inviteAlreadyInQueue.message = Please leave all queues before inviting a player
-teammatchmaking.notification.joinAlreadyInQueue.title = You are already in a queue
-teammatchmaking.notification.joinAlreadyInQueue.message = Please leave all queues before joining a party
-teammatchmaking.notification.customAlreadyInQueue.title = You are already in a matchmaking queue
-teammatchmaking.notification.customAlreadyInQueue.message = Please leave all queues before joining a custom game
+teammatchmaking.notification.inviteAlreadyInQueue.title = You are already searching for a game
+teammatchmaking.notification.inviteAlreadyInQueue.message = Please stop searching for a game before inviting a player
+teammatchmaking.notification.joinAlreadyInQueue.title = You are already searching for a game
+teammatchmaking.notification.joinAlreadyInQueue.message = Please stop searching for a game before joining a party
+teammatchmaking.notification.customAlreadyInQueue.title = You are already searching for a game
+teammatchmaking.notification.customAlreadyInQueue.message = Please stop searching for a game before joining a custom game
 teammatchmaking.notification.gameAlreadyRunning.title = Already in game
-teammatchmaking.notification.gameAlreadyRunning.message = You cannot join a queue while you're in game
+teammatchmaking.notification.gameAlreadyRunning.message = You cannot join matchmaking while you're in game
 teammatchmaking.playerTitle = Playing matchmaking as\:
 teammatchmaking.partyTitle = Your party\:
-teammatchmaking.queueTitle = Join one or more queues when you are ready\:
-teammatchmaking.queueTitle.inGame = Can't join queues while you are in a game
-teammatchmaking.queueTitle.memberInGame = Can't join queues while someone is in a game
-teammatchmaking.queueTitle.inParty = The party leader will join the queues
-teammatchmaking.queueTitle.inQueue = Your party is now in queue
+teammatchmaking.searchButton = Play Forged Alliance Forever
+teammatchmaking.searchButton.inGame = Can't join matchmaking while you are in a game
+teammatchmaking.searchButton.memberInGame = Can't join matchmaking while someone is in a game
+teammatchmaking.searchButton.inParty = The party leader will join the matchmaking
+teammatchmaking.searchButton.inQueue = Your party is now searching for a game
```

Comment:
There isn't really a reason to change the key name I. Doing so just breaks translations that already exists

Reply:
We are also changing the text here. If we don't change the key the old translations will still apply and the users of the client won't even notice that a retranslation is necessary

Reply:
Perhaps although the rewording seems pretty minimal on all but the first I question if it is truly necessary